api/staff-ref-data/mock-staff-ref-data-api.ts

```diff
@@ -91,9 +91,8 @@ export class MockStaffRefDataAPI implements StaffRefDataAPI {
     }
   }
 
-  async getStaffRefUserDetails(req, res: Response, next: NextFunction) {
-    const id = req.params.id;
-    const apiPath: string = `/refdata/case-worker/user-details/${id}`;
+  async getStaffRefUsersById(req, res: Response, next: NextFunction) {
```

Comment:
Why is this getStaffRefUsersById if there isn't an ID?

Reply:
We need to pass in an array of email_ids. it's in req.body